apps/files_sharing/src/components/SharingEntry.vue

```diff
@@ -29,17 +29,24 @@
 		<div v-tooltip.auto="tooltip" class="sharing-entry__desc">
 			<h5>{{ title }}</h5>
 		</div>
+		<!-- edit permission -->
+		<ActionCheckbox
+			ref="canEdit"
+			:checked.sync="canEdit"
+			:value="permissionsEdit"
+			:disabled="saving">
+			{{ t('files_sharing', 'Can edit') }}
```

Comment:
```suggestion
			{{ t('files_sharing', 'Allow editing) }}
```

I actually thing it's more clear to have the wording "allow" as it's clearly stating a permission, while "can xxx seems a bit unsettling)
cc @jancborchardt our wording ux expert ;)

Reply:
+1

Reply:
Yes, "Can edit" seems a bit pedestrian. Also doesn’t work that well with the language below for "Set expiration date".

- Allow editing
- Allow creation
- Allow resharing
- Allow deletion